collectors/python.d.plugin/pandas/metadata.yaml

```diff
@@ -0,0 +1,310 @@
+
+      This collector can be used to collect pretty much anything that can be read by Pandas, and then processed by Pandas.
+
+      More detailed information can be found in the Netdata documentation [here](https://learn.netdata.cloud/docs/data-collection/generic-data-collection/structured-data-pandas).
+    method_description: |
+      The collector uses [pandas](https://pandas.pydata.org/) to pull data and do pandas-based preprocessing, before feeding to Netdata.
+  supported_platforms:
+    include: []
+    exclude: []
+  multi_instance: true
+  additional_permissions:
+    description: ""
+  default_behavior:
+    auto_detection:
+      description: ""
+    limits:
+      description: ""
+    performance_impact:
+      description: ""
+setup:
+  prerequisites:
+    list:
+      - title: Python Requirements
+        description: |
+          This collector depends on some Python (Python 3 only) packages that can usually be installed via `pip` or `pip3`.
+
+          ```bash
+          sudo pip install pandas requests
+          ```
+
+          Note: If you would like to use [`pandas.read_sql`](https://pandas.pydata.org/docs/reference/api/pandas.read_sql.html) to query a database, you will need to install the below packages as well.
+
+          ```bash
+          sudo pip install 'sqlalchemy<2.0' psycopg2-binary
+          ```
+  configuration:
+    file:
+      name: python.d/pandas.conf
+      description: ""
+    options:
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.
+
+        Additionally, the following collapsed table contains all the options that can be configured inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
+      folding:
+        title: Config options
+        enabled: true
+      list:
+        - name: chart_configs
+          description: an array of chart configuration dictionaries
+          default_value: []
+          required: true
+        - name: chart_configs.name
+          description: name of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.title
+          description: title of the chart to be displayed in the dashboard.
+          default_value: None
+          required: true
+        - name: chart_configs.family
+          description: "[family](https://learn.netdata.cloud/docs/data-collection/chart-dimensions-contexts-and-families#family) of the chart to be displayed in the dashboard."
```

Comment:
same here, I would rather link to github to have a static source to link to

Reply:
can clean up later